src/Plugin/Field/FieldType/OgGroupItem.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\og\OgGroupItem.
+ */
+
+namespace Drupal\og\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\TypedData\DataDefinition;
+
+/**
+ * OG Group Data class to return TRUE always.
+ *
+ * This field type is used simply as a placeholder for a field formatter that
+ * would allow non-groups members, members, and group managers to join, request
+ * or leave a group.
+ *
+ * @FieldType(
+ *   id = "og_group",
+ *   label = @Translation("OG Group"),
+ *   description = @Translation("OG Group"),
+ *   category = @Translation("OG"),
+ *   no_ui = TRUE,
+ *   default_formatter = "og_ui_group_subscribe",
+ * )
+ */
+class OgGroupItem extends FieldItemBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function schema(FieldStorageDefinitionInterface $field_definition) {
+    return [];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function propertyDefinitions(FieldStorageDefinitionInterface $field_definition) {
+    $definitions = [];
+
+    $definitions['value'] = DataDefinition::create('boolean')
+      ->setLabel('Group item value')
+      ->setReadOnly(TRUE)
+      ->setComputed(TRUE)
+      ->setClass('\Drupal\og\OgGroupBoolean')
+      ->setSetting('group_value', TRUE);
```

Comment:
Don't think I really like this approach. Was just because it is in theory most flexible for the future. Not sure we would ever need to change this though? If not, I'll just hardcode the value to TRUE again in the `OgGroupBoolean` class.